wework/scripts/dev-executor-sidecar.sh

```
@@ -14,7 +14,38 @@ if [ "${WEGENT_CARGO_TARGET_DIR_AUTO:-0}" = "1" ]; then
+    configure_wegent_cargo_target_dir "$PROJECT_DIR" "executor"
+    return 0
+  fi
+
+  local cache_root
+  cache_root="$(wegent_cargo_cache_root)"
+  if [ -z "$cache_root" ]; then
+    configure_wegent_cargo_target_dir "$PROJECT_DIR" "executor"
+    return 0
+  fi
+
+  export CARGO_TARGET_DIR="$cache_root/executor-dev/$(executor_source_cache_key)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)